### DISCUSSION TITLE
Requesting help on the Wifi FTM example

### DISCUSSION THREAD

**Discussion by JJ8428** [espressif/arduino-esp32#8476] [Q&A]
Hello, this is my first time creating a discussion on Github, so I hope I am not breaking any formatting protocol.

I am trying to recreate the Wifi FTM example that can be found at the following link: https://github.com/espressif/arduino-esp32/tree/master/libraries/WiFi/examples/FTM. I have on myself 2 ESP32s2 and I am using the Arduino IDEs. I'm able to upload basic scripts with ease, however, I have been unable to solve this problem with the FTM example. 

On one board, I have the FTM responder code set up to host a SoftAP. On the other board, I have the code for a FTM initiator. When I run this code, the FTM works fine for a short period of time, however after a short duration of time has passed, I get a CONF_REJECTED error. 

From my research, it looks like the SoftAP hosted by the FTM responder has some data limit, and once that data limit is reached, the responder kicks the FTM intiator off of it, resulting in the CONF_REJECTED. 

Below is what the serial monitor connected to the FTM initiator prints out. 

FTM Estimate: Distance: 2.34 m, Return Time: 15 ns
FTM Estimate: Distance: 2.21 m, Return Time: 14 ns
FTM Estimate: Distance: 2.08 m, Return Time: 13 ns
FTM Estimate: Distance: 2.21 m, Return Time: 14 ns
FTM Estimate: Distance: 2.08 m, Return Time: 13 ns
FTM Estimate: Distance: 2.08 m, Return Time: 13 ns
FTM Estimate: Distance: 2.21 m, Return Time: 14 ns
FTM Estimate: Distance: 2.21 m, Return Time: 14 ns
FTM Estimate: Distance: 2.34 m, Return Time: 15 ns
FTM Estimate: Distance: 1.82 m, Return Time: 12 ns
FTM Estimate: Distance: 1.69 m, Return Time: 11 ns
FTM Error: CONF_REJECTED

I browsed the web for hours but cannot find a solution and I have posted on the espressif forums and reddit to no avail. Some additional details are that I both ESP32s connected to the same laptop (on different ports). and that the code is a carbon copy of that found in the repo listed above.

Any suggestions as to what I should do? If any additional information is needed, I will happily provide.

Some additional context is that I am rather keen on using the Arduino IDE. I am working on a project that focuses on in door localization using another Arduino Board and I want to keep the number of coding platforms I use to a minimum to make the project documentation as simple as possible so any future expansions of it are not split between ESP-IDF and Arduino IDE.

**Comment by geofrancis** [comment]
I am getting the same result when testing with a pair of Wemos S2 mini boards 

WiFi Connected
Initiating FTM session with Frame Count 16 and Burst Period 200 ms
FTM Estimate: Distance: 0.26 m, Return Time: 1 ns    
FTM Estimate: Distance: 1.30 m, Return Time: 8 ns    
FTM Estimate: Distance: 0.91 m, Return Time: 6 ns   
FTM Estimate: Distance: 1.04 m, Return Time: 6 ns   
FTM Estimate: Distance: 0.78 m, Return Time: 5 ns  
FTM Estimate: Distance: 0.52 m, Return Time: 3 ns   
FTM Estimate: Distance: 0.65 m, Return Time: 4 ns    
FTM Estimate: Distance: 0.52 m, Return Time: 3 ns    
FTM Estimate: Distance: 0.39 m, Return Time: 2 ns   
FTM Estimate: Distance: 1.04 m, Return Time: 6 ns   
FTM Estimate: Distance: 0.52 m, Return Time: 3 ns    
FTM Estimate: Distance: 1.30 m, Return Time: 8 ns   
FTM Estimate: Distance: 0.39 m, Return Time: 2 ns   
FTM Estimate: Distance: 3.38 m, Return Time: 22 ns    
FTM Estimate: Distance: 2.60 m, Return Time: 17 ns
FTM Estimate: Distance: 0.00 m, Return Time: 0 ns     
FTM Estimate: Distance: 1.17 m, Return Time: 7 ns   
FTM Estimate: Distance: 0.00 m, Return Time: 0 ns    
FTM Estimate: Distance: 0.78 m, Return Time: 5 ns     
FTM Estimate: Distance: 0.00 m, Return Time: 0 ns     
FTM Estimate: Distance: 1.17 m, Return Time: 7 ns     
FTM Estimate: Distance: 2.21 m, Return Time: 14 ns     
FTM Error: CONF_REJECTED

**Comment by geofrancis** [comment]
increasing the burst period seems to help. it only happens regularly at the stock speed.

**Comment by JJ8428** [comment]
Hey thank you for responding. 

I ended up actually going with the ESP-IDF platform instead. It was a pain to learn to use that instead of the beloved Arduino IDE.
